test cases/common/58 run target/fakeburner.py

```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
+
+from __future__ import print_function
 
```

Comment:
Don't need to do this import if we'll use the python3 module to run it with python3.